CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Bolt NEXT
 
+### Deprecations
+
+* **WARNING**: Support for plan method `Target.new(<uri>, <options>)` will be dropped in the near future. The plan method `Target.new(<uri>)` should be used instead.
```

Comment:
I think we want users to use an options hash, like `Target.new('uri' => <uri>, 'config' => { 'ssh' => { 'port' => '23' } })`